src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.cs

```diff
@@ -20,6 +20,7 @@ public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, S
         public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object?>> observer, System.Predicate<string>? isEnabled) { throw null; }
         public override string ToString() { throw null; }
         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("The type of object being written to DiagnosticSource cannot be discovered statically.")]
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("The object parameter might contain an instance of a ValueType that might not be available during runtime")]
```

Comment:
If I have an instance of something at runtime, how can it be unavailable at the same time?